icu4c/source/data/buildtool/renderers/__init__.py

```diff
@@ -0,0 +1,10 @@
+# Copyright (C) 2018 and later: Unicode, Inc. and others.
+# License & terms of use: http://www.unicode.org/copyright.html
+
+from collections import namedtuple
+
+MakeRule = namedtuple("MakeRule", ["name", "dep_literals", "dep_files", "output_file", "cmds"])
```

Comment:
I have one note from our Thursday session that I don't remember what specific thing we were looking at...

The note says "alternative: compare time stamps in Python" -- which I think refers to the discussion of generating a makefile vs. just doing everything directly, including sometimes skipping tools when the input is older than the output. (Except need to regenerate when config changes etc.)

I don't remember if this came up here or for "windirect" :-/ Do you remember? Was it just to add a note/comment somewhere?

Reply:
I added a comment in makedata.mak